core/src/main/scala/org/ensime/config/EnsimeConfigProtocol.scala

```diff
@@ -28,17 +27,18 @@ object EnsimeConfigProtocol {
   private implicit val moduleFormat: SexpFormat[EnsimeModule] = cachedImplicit
   private implicit val configFormat: SexpFormat[EnsimeConfig] = cachedImplicit
 
-  def parse(config: String): EnsimeConfig = {
+  def parse(config: String, serverConfig: String): EnsimeConfig = {
```

Comment:
@yyadavalli we don't need to write a parser, we can just use a minimal shapeless impl and typesafe config. We don't need to support sealed traits or anything at this stage just an extremely small subset of https://github.com/melrief/pureconfig